src/dailyai/transports/daily_transport.py

```diff
@@ -268,6 +274,37 @@ def dialout(self, number):
     def start_recording(self):
         self.client.start_recording()
 
+    def render_participant_video(self,
+                                 participant_id,
+                                 framerate=10,
+                                 video_source="camera",
+                                 color_format="RGB") -> None:
+        if not self._video_rendering_enabled:
+            self._logger.warn("Video rendering is not enabled")
+
+        self._video_renderers[participant_id] = {
+            "framerate": framerate,
+            "timestamp": 0,
+        }
+        self.client.set_video_renderer(
+            participant_id,
+            self.on_participant_video_frame,
+            video_source=video_source,
+            color_format=color_format)
+
+    def on_participant_video_frame(self, participant_id, video_frame):
+        curr_time = time.time()
+        prev_time = self._video_renderers[participant_id]["timestamp"]
+        diff_time = curr_time - prev_time
+        period = 1 / self._video_renderers[participant_id]["framerate"]
```

Comment:
One thing I know we'll need eventually is an "only on demand" option instead of always setting a frame rate. My way of doing that was maybe a bit clunky but it worked well—by setting the frame rate to 0, the transport would never send frames automatically, but the pipeline could request a new image for a participant by sending a certain frame type (`RequestUserImage` or something). Can we add that back in a follow-up PR?